Sleep Tracker/Sleep Tracker/View/SleepPageView/SleepSetupView/SleepSetupView.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+/// A view show the option to set the alarm clock and set the wake-up time before go to bed.
```

Comment:
nit: 

Maybe that would be better?

```suggestion
/// A view shows the options that the user can set before go to bed.
```

Reply:
I agree, fixed